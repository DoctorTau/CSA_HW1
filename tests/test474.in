623
-6657 -5442 6481 7639 5337 -8991 -6911 -4206 4630 172 -4328 -4284 9981 -6573 1232 1894 5812 -296 9244 8229 -4552 1562 -5609 -221 -8828 -5122 -3660 -3760 -929 -742 -9487 68 9300 -267 4631 -6903 -2281 1641 4489 -705 -5249 -414 -5793 6862 -4807 8302 4905 7955 -7568 -2071 -6633 -1345 6901 -5511 -4624 7955 8363 6531 5364 1275 -154 5123 -2357 8943 5768 -6615 -7949 6928 4142 -4960 1441 930 4903 -6365 -8820 -1419 -7911 3997 -6465 -8205 -5587 -8254 -6860 6745 1667 -310 -4852 -3399 -6143 -1991 -2833 7147 -6919 5225 -9995 2498 -1728 -4694 7507 -4966 -393 6704 -9538 -4780 -1116 5277 -355 2933 -8337 -1680 8381 -9695 -3475 -2226 3222 9631 6232 3012 8242 3352 -9876 -8737 5080 -3107 3761 5546 -3140 4362 8609 -6176 -6046 4538 -8269 -9976 -6560 99 7236 2161 -3696 -1531 -6858 -3286 8143 -1609 2601 7983 643 2066 4556 5589 -1969 -1344 -5269 4728 9990 7288 2229 -9111 6471 1394 -9399 2966 899 5898 6492 4850 -1795 -9829 -9450 9781 -3137 -7122 6288 -598 -1355 -1489 6311 -2630 6504 11 3829 -7422 -3500 7826 -7661 -3558 -9722 -1501 4475 7957 -5392 371 -8796 -3622 -2802 -7777 6920 679 -9508 -876 6568 -2823 4645 -7843 5424 4578 -2138 -3329 -5669 8987 8069 -1637 -2506 671 -2019 -3316 2803 -1561 1813 -565 -9827 5265 8056 7021 -9103 -1168 -5715 4913 -2688 3788 4526 2929 2544 6374 -3361 -2121 6082 7208 -3144 -7476 159 -9184 -9562 8447 -6188 -7166 9039 -6569 9988 3278 2787 6766 1464 3459 544 -6942 -7820 7117 -4043 -598 7079 -4281 1104 1019 -1422 3070 -7509 -5081 6807 -4743 -4050 1098 -3731 6785 4629 1840 2350 6111 -4938 -7298 4893 3108 -4710 -3768 -569 -4687 2962 -561 -3454 -3935 2637 -1238 6564 4979 922 -5382 7514 -6936 -1309 2379 3941 1559 7291 -1119 3273 3999 -4254 -9222 346 9771 -2209 8018 -843 8905 6358 -8664 5539 -4047 3407 9589 -3440 -6273 3455 -8536 2256 164 -2133 8332 7328 6913 5410 6900 -8961 -6853 -2065 5791 -8498 9565 8800 -1890 -5148 -8184 9482 -2066 -4362 5603 2545 -9190 4376 1856 9073 -2100 426 6720 594 -8803 735 -6169 7346 6049 -3017 9584 6151 1802 1399 -8079 4581 -1919 5501 -1504 313 4496 -9695 -4714 -8959 -2669 8670 -5990 -2345 -7786 -2531 -679 368 5203 671 9864 -4368 -8518 -5733 -8714 -2073 -9855 -9261 -8821 4914 -9710 -439 5777 -1953 -9720 4324 -1042 -4863 3560 -1822 -8556 -1955 6453 5176 2335 5646 -9697 -8431 4539 314 -6466 -6779 6343 9939 -8333 2064 -5098 7220 117 -8751 -5709 452 -9769 -9476 170 4261 -5629 4265 1672 -2996 681 -8841 -65 1735 -9462 8866 -8796 -6688 -7503 -4779 -5135 -4292 19 2486 -8269 4010 6965 -1479 -5596 8656 9710 2576 -3424 1492 -3678 1540 3133 5581 3507 -1312 4338 -2049 2158 5357 4217 -2815 7985 -3543 -7591 -813 4435 9480 2160 7495 5629 9117 -8932 1836 -6935 3653 -7377 2054 -4518 -9745 8455 7814 -9046 4236 2807 -5231 -3392 -9553 -1781 -4232 -9285 2106 7742 6932 -3611 5376 1280 -862 9764 5381 -3680 -3823 5519 6837 -8144 -2971 106 -2374 -6564 -141 -4152 7333 -6320 3894 -1310 114 1579 27 3303 6107 6327 8513 4303 2639 1039 8572 -8411 5338 5407 -7975 -4955 -6082 4322 8671 1838 -9473 -9976 5733 -2953 2921 1733 -8018 -4144 -2922 -4839 9584 -6147 7564 8990 1028 9133 298 9479 969 -908 4238 -2503 2293 2940 2920 -2562 -9622 2873 3607 -3218 -8696 9260 -6509 4317 9120 -8211 -8051 -2206 9201 -3274 4977 4560 320 -5204 7336 -5262 9445 2759 3880 -9416 3252 -6828 -5943 7693 7370 6220 1844 -9287 5625 -5487 -4462 -1066 -3596 850 -3782 8602 1410 -99 2697 2340 -5313 7753 -1818 836 -2056 11 1750 -8641 -2078 