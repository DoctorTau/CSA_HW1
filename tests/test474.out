-5442 -4206 -4328 -4284 -296 -4552 -8828 -5122 -3660 -3760 -742 -414 -7568 -4624 -154 -4960 -8820 -8254 -6860 -310 -4852 -1728 -4694 -4966 -9538 -4780 -1116 -1680 -2226 -9876 -3140 -6176 -6046 -9976 -6560 -3696 -6858 -3286 -1344 -9450 -7122 -598 -2630 -7422 -3500 -3558 -9722 -5392 -8796 -3622 -2802 -9508 -876 -2138 -2506 -3316 -1168 -2688 -3144 -7476 -9184 -9562 -6188 -7166 -6942 -7820 -598 -1422 -4050 -4938 -7298 -4710 -3768 -3454 -1238 -5382 -6936 -4254 -9222 -8664 -3440 -8536 -8498 -1890 -5148 -8184 -2066 -4362 -9190 -2100 -1504 -4714 -5990 -7786 -4368 -8518 -8714 -9710 -9720 -1042 -1822 -8556 -6466 -5098 -9476 -2996 -9462 -8796 -6688 -4292 -5596 -3424 -3678 -1312 -8932 -4518 -9046 -3392 -4232 -862 -3680 -8144 -2374 -6564 -4152 -6320 -1310 -6082 -9976 -8018 -4144 -2922 -908 -2562 -9622 -3218 -8696 -2206 -3274 -5204 -5262 -9416 -6828 -4462 -1066 -3596 -3782 -1818 -2056 -2078 